### ISSUE TITLE
Make Jenkins yell about whitespace issues

### ISSUE THREAD

**Issue by remexre** [melt-umn/silver#526]
e.g. using tabs in `.sv` (and `.sv.md`? just in silver blocks there?) files, not having trailing newlines in the same

**Comment by ericvanwyk** [comment]
This seems reasonable but what is the motivation for doing this? Is it a problem now?

**Comment by remexre** [comment]
Ended up finding a bunch of it in the initial docgen PR, seems like it'd save some time if something other than human review would find it, since it's trivially machine-checkable

**Comment by ericvanwyk** [comment]
Yes but what is the problem? Do markdown not render well with these?

**Comment by remexre** [comment]
well, my editor has tabs = 8 spaces, and louis' had tabs = 4... and the editor introduces things like `\t\t  foo.bar = baz;`, which is just ugly

trailing newline is just unix convention, which... sublime text, I think he uses? wasnt' respecting

**Comment by ericvanwyk** [comment]
OK - that makes sense. Let's do this.

**Comment by krame505** [comment]
Isn't there a nicer way now to set this up as a check on github?  I would prefer to not have Jenkins builds failing due to formatting issues, and it would be good to avoid giving Jenkins additional sorts of responsibilities if we think we may want to replace it at some point.
